tei/poliziano-orfeo.xml

```diff
@@ -388,7 +396,9 @@
             <l>a me basta guardar le vacche e ' buoi.</l>
           </lg>
         </sp>
-        <stage>Aristeo ad Euridice:</stage>
+        <sp who="#aristeo">
+          <speaker>Aristeo:</speaker>
```

Comment:
The one after "Euridice" was already in the source. I probably added `<speaker>Aristeo:</speaker>` by mistake, since it is not in the source and should be omitted (provided one keeps the who-tag to link the speech to the character). Thanks!